seqr/views/apis/saved_variant_api.py

```diff
@@ -137,26 +137,20 @@ def _create_variant_note(saved_variants, note_json, user, genome_version):
 
 
 def _variant_gene_id(variant, genome_version):
-    main_transcript_id = variant.selected_main_transcript_id or variant.saved_variant_json.get('mainTranscriptId')
-    if main_transcript_id:
+    if variant.selected_main_transcript_id and len(variant.gene_ids) > 1:
```

Comment:
gene_ids are sorted so the gene associated with the `mainTranscriptId` is always first, so there is no longer any need to determine which gene is the "main" one